packages/ibm-products/src/components/EmptyStates/assets/NotificationsIllustration.js

```diff
@@ -33,6 +33,7 @@ export const NotificationsIllustration = ({ theme, size, alt, ...rest }) => {
         `${blockClass}__illustration--${size}`,
       ])}
       role="img"
+      aria-label="Illustration of an empty notification state"
```

Comment:
Rather than hardcoding the `aria-label`, we can probably either reuse the `alt` in L38 or use `aria-labelledby`.

Reply:
oh that showed me we just actually needed something in the `title`, so we can use `illustrativeDescription` and it also solved the fix